### PR TITLE
Fixes some bugs

### PR DESCRIPTION
**Pull request by Graphcalibur**
Tower Preview has been disabled when the player hasn't seen any enemies yet. Also fixed a bug where saving in one of the editors crashed the game.

### CODE REVIEW

**Review by vincetiu8 — CHANGES_REQUESTED** [review]
Tower preview was disabled intentionally as it crashes if the player hasn't seen any enemies: to preview the tower, enemies need to spawn...

**Comment by Graphcalibur** [comment]
Ah, I wasn't aware that's already been fixed. Merge conflicts should be resolved now.

**Comment by vincetiu8** [comment]
Oh whooops, I thought you were trying to revert that change.

File: data/menus.py [hunk 1]
```diff
@@ -219,10 +221,17 @@ def event(self, event):
                     if len(SAVE_DATA["seen_enemies"]) > 0:
                         BTN_SFX.play()
                         return "tower_preview"
+<<<<<<< HEAD
```
Comment (vincetiu8) [inline data/menus.py]: This causes the game to crash

**Review by vincetiu8 — CHANGES_REQUESTED** [review]

**Comment by Graphcalibur** [comment]
Should be fixed now @Invincibot

**Review by vincetiu8 — APPROVED** [review]
LGTM!

**Pull request merged**